mscore/scoreview.cpp

```diff
@@ -3613,12 +3613,6 @@ void ScoreView::contextPopup(QContextMenuEvent* ev)
                   objectPopup(gp, e);
                   }
             }
-      else {
```

Comment:
Try it. I will do a pass on it anyway since I'm modifying exactly the same files to make translation of the menus dynamic when the user changes the translation...